### PR TITLE
Components Cheatsheet- InputFamily

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Refactoring Components Cheatsheet.
Splitting to multiple files and implementing InputFamily as described in the spec.

Zeplin: https://zpl.io/2pMwZJr
Google doc: https://docs.google.com/spreadsheets/d/10-wyY2uc80fknWcLGpQ-nCQ94q5kmmEYfAbppQ63wEg

### CODE REVIEW

File: stories/Introduction/ButtonFamily.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (shlomitc) [inline stories/Introduction/ButtonFamily.js]: please move all of these files to a new folder called `Cheatsheet` (nested in `Introduction`)

File: stories/Introduction/Cheatsheet/ComponentsCheatsheet.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+//importing components by family type
+import FoundationFamily from './ComponentsFamilies/FoundationFamily';
+import LayoutFamily from './ComponentsFamilies/LayoutFamily';
+import InputFamily from './ComponentsFamilies/InputFamily';
+import SelectionFamily from './ComponentsFamilies/SelectionFamily';
+import ButtonFamily from './ComponentsFamilies/ButtonFamily';
+import NavigationFamily from './ComponentsFamilies/NavigationFamily';
+import TooltipFamily from './ComponentsFamilies/TooltipFamily';
+import NotifiationFamily from './ComponentsFamilies/NotificationFamily';
```
Comment (sidoruk-sv) [inline stories/Introduction/Cheatsheet/ComponentsCheatsheet.js]: NotificationFamily

File: stories/Introduction/Cheatsheet/ComponentsFamilies/NotificationFamily.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { GeneralStructure, SingleComponent } from '../structure';
+
+//8. Notifications
+import Notification from 'wix-style-react/Notification';
+import FloatingNotification from 'wix-style-react/FloatingNotification';
+import SectionHelper from 'wix-style-react/SectionHelper';
+
+const NotifiationFamily = () => (
```
Comment (sidoruk-sv) [inline stories/Introduction/Cheatsheet/ComponentsFamilies/NotificationFamily.js]: typo: should be Notifi**c**ationFamily

File: stories/Introduction/Cheatsheet/structure.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <Card.Header title={title} />
+    <Card.Content>
+      <Container fluid>
+        <SingleComponent
+          name={
+            <Heading appearance="H5" light>
+              INDEX NAME & I.C.
+            </Heading>
+          }
+          children={
```
Comment (shlomitc) [inline stories/Introduction/Cheatsheet/structure.js]:
```jsx
<SingleComponent>
  <Heading appearance="h5" light>
    Preview
  </Heading>
<SingleComponent>
```

File: stories/Introduction/Cheatsheet/structure.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import Box from 'wix-style-react/Box';
+
+export const SingleComponent = ({
+  name,
+  componentName,
+  children,
+  secondary,
+  light,
+}) => {
+  debugger;
```
Comment (shlomitc) [inline stories/Introduction/Cheatsheet/structure.js]: remove

File: stories/Introduction/Cheatsheet/structure.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  //TODO: should check if title or child component
+
+  return (
+    <Row>
+      <Col span={6}>
+        <Box>
```
Comment (shlomitc) [inline stories/Introduction/Cheatsheet/structure.js]: marginBottom={'6px'}

File: stories/Introduction/Cheatsheet/structure.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+          <Text secondary={secondary} light={light} size="medium" weight="bold">
+            {name}
+          </Text>
+        </Box>
+        <Box>{componentName}</Box>
```
Comment (shlomitc) [inline stories/Introduction/Cheatsheet/structure.js]: content should be wrapped with `Text` component (small, thing, secondary)

File: stories/Introduction/Cheatsheet/structure.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+import { Container, Row, Col } from 'wix-style-react/Grid';
+import Card from 'wix-style-react/Card';
+import Text from 'wix-style-react/Text';
+import Heading from 'wix-style-react/Heading';
+import Box from 'wix-style-react/Box';
+
+export const SingleComponent = ({
+  name,
+  componentName,
```
Comment (shlomitc) [inline stories/Introduction/Cheatsheet/structure.js]: should be `componentsNames` and it should be an array of items.
for example `['<FormField/>',  '<Input/>]`

File: stories/Introduction/Cheatsheet/sharedComponents.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
Comment (shlomitc) [inline stories/Introduction/Cheatsheet/sharedComponents.js]: merge to the same file of reuseable helper components

**Pull request merged**